### ISSUE TITLE
[FR] Guest access

### ISSUE THREAD

**Issue by Weissnix4711** [ajbura/cinny#1161]
### Describe the problem

It would be nice to allow users to preview rooms without signing in.

### Describe the solution you'd like

A solution similar to what element web does right now, where users can preview any public room which has been made available to anonymous users, though not necessarily join (but that would be a nice addition too).

**Comment by Cadair** [comment]
Would also be nice to be able to preview a space in a similar way.

**Comment by tezlm** [comment]
Note that guest accounts and peeking are different things in the spec, where the latter isn't in the spec yet ([msc2753](https://github.com/matrix-org/matrix-spec-proposals/pull/2753), [msc2444](https://github.com/matrix-org/matrix-spec-proposals/pull/2444)) (element does this by using deprecated apis that some servers (ie conduit) don't support)

Having guest accounts but no room peeking is somewhat useless so I'd say this feature is blocked by spec for now?

**Comment by paboum** [comment]
No, it doesn't mean feature is blocked, only that it will perhaps require some reworking after spec will be finished. The goal here should be implementing the feature in a way that will be easy to update with new api in the future, not to postpone it indefinitely until the client gives up.
